### PR TITLE
add a UI page with options

### PR DESCRIPTION
**Pull request by wangzz2019**
Add a UI Page as options that could help people setting RUM options easily

### CODE REVIEW

File: options.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head><title>My Test Extension Options</title></head>
```
Comment (shigeya-dd) [inline options.html]: Would you update the title of this page?
```suggestion
<head><title>RUM Demo Extension Options</title></head>
```

**Comment by shigeya-dd** [comment]
LGTM Thanks!

**Review by shigeya-dd — CHANGES_REQUESTED** [review]
Would you update the title of the option page? Thanks!

File: README.md [hunk 2]
```diff
@@ -27,3 +27,10 @@
   - _RUM Demo_ will appear in the extensions page and the extensions menu.
 - When you modified files in this folder, they will be reloaded in Chrome automatically.
   - It's done by hot-reload.js. [Author's repo](https://github.com/xpl/crx-hotreload)
+
+- Options: You can also set datadog tags from options page.
+  - Pin it on extension bar
+  - ![pin](https://a.cl.ly/12uvmjJQ)
```
Comment (shigeya-dd) [inline README.md]: Screenshots looks not working..

**Comment by wangzz2019** [comment]
updated and add some instructions in README

File: README.md (hunk 2 above)
Comment (wangzz2019) [reply README.md]: It should work now

**Comment by shigeya-dd** [comment]
LGTM 😃

**Review by shigeya-dd — APPROVED** [review]

**Pull request merged**